examples/compute_plan/scripts/add_compute_plan.py

```diff
@@ -26,8 +26,7 @@
 with open(os.path.join(current_directory, '../../config.json'), 'r') as f:
     config = json.load(f)
```

Comment:
This code can be removed (and the config.json file could also be removed).

Reply:
That's right 👍